### ISSUE TITLE
path to xrdp.ini and sesman.ini as command line parameter

### ISSUE THREAD

**Issue by volth** [neutrinolabs/xrdp#609]
Please add possibility to specify path to ```xrdp.ini``` and ```sesman.ini``` in command-line of ```xrdp``` and ```xrdp-sesman``` (e.g. ```xrdp --conf=/somepath/xrdp.ini```).

It would allow to port ```xrdp``` to [nixos](http://nixos.org).

**Comment by proski** [comment]
Good idea. It would also allow running multiple versions of xrdp and xrdp-sesman on the same system (without using chroot).

**Comment by metalefty** [comment]
+1

**Comment by speidy** [comment]
+1

**Comment by volth** [comment]
rebased to ```0.9.2```: https://github.com/volth/xrdp/tree/runtime-cfg-path-0.9.2

**Comment by volth** [comment]
rebased to ```0.9.3``` https://github.com/neutrinolabs/xrdp/compare/devel...volth:runtime-cfg-path-0.9.3

**Comment by volth** [comment]
rebased to ```0.9.5``` https://github.com/neutrinolabs/xrdp/compare/v0.9.5...volth:runtime-cfg-path-0.9.5

**Comment by metalefty** [comment]
I'll test this.

**Comment by volth** [comment]
rebased to ```0.9.7``` https://github.com/neutrinolabs/xrdp/compare/v0.9.7...volth:runtime-cfg-path-0.9.7
